content/rc/subscriptions/create-fixed-subscription.md

```diff
@@ -52,13 +52,13 @@ To create a [Fixed subscription]({{< relref "/rc/subscriptions/#subscription-pla
 
 3.  Choose a **Cloud Provider** and a **Region**.
 
-    {{<image filename="images/rc/subscription-new-cloud-vendor-options-redis-7-preview.png" alt="Available cloud vendor options and Redis 7 preview." >}}{{</image>}}
+    {{<image filename="images/rc/subscription-new-cloud-vendor-options-redis-7-preview.png" alt="Available cloud vendor options and Redis 7.2 preview." >}}{{</image>}}
 
-    A [preview of Redis 7.0]({{<relref "/rc/changelog/march-2023#redis-70-preview">}}) subscriptions is available for selected regions in AWS and GCP. However, some Redis 7.0 functionality might not be fully available during preview. Redis 7.0 also introduces several changes to existing Redis commands; see the list of [breaking changes]({{<relref "/rc/changelog/march-2023#redis-70-breaking-changes">}}) for more details.
+    A [preview of Redis 7.2]({{<relref "/rc/changelog/june-2023#redis-72-preview">}}) subscriptions is available for selected regions in AWS and GCP. Redis 7.2 introduces several changes to existing Redis commands; see the list of [breaking changes]({{<relref "/rc/changelog/june-2023#redis-72-breaking-changes">}}) for more details.
     
-    If you want to try out Redis 7.0, turn on the **Redis 7.0 preview** toggle:
+    If you want to try out Redis 7.2, turn on the **Redis 7.2 preview** toggle:
 
-    {{<image filename="images/rc/subscription-new-redis-7-preview-toggle.png" width="200px" alt="Turn on the Redis 7 preview toggle." >}}{{< /image >}}
+    {{<image filename="images/rc/subscription-new-redis-7-preview-toggle.png" width="200px" alt="Turn on the Redis 7.2 preview toggle." >}}{{< /image >}}
```

Comment:
very small thing, the screenshot has pixels at the right, maybe we want to keep it with ℹ️